src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IClaimingManager.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+interface IPaymentCoordinator {
+
+    /// STRUCTS ///
+	struct RangePayment {
+        address avs;
+		IStrategy strategy;
+		IERC20 token;
+		uint256 amount;
+		uint256 startRangeTimestamp;
+		uint256 endRangeTimestamp;
+        bool toOperatorSet;
+	}
+
+    /// EVENTS ///
+
+    event RangePaymentCreated(RangePayment rangePayment);
+
+    /// VIEW FUNCTIONS ///
+
+    /// @notice The address of the entity that can update the contract with new merkle roots
+    function claimingManager() external view returns (IClaimingManager);
```

Comment:
I think this comment is inaccurate?
the usage here would be the address where payments are forwarded, right?